### ISSUE TITLE
FAILED: StreamTestKitSpec TestSink Probe

### ISSUE THREAD

**Issue by ktoso** [akka/akka#19464]
https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/4828/console

```
[info] A TestSink Probe
[info] - must #toStrict (172 milliseconds)
[info] - must #toStrict with failing source *** FAILED *** (47 milliseconds)
[info]   "toStrict received OnError while draining stream! Accumulated elements: List(1, 2)" did not include substring "Boom!" (StreamTestKitSpec.scala:40)
[info]   org.scalatest.exceptions.TestFailedException:
[info]   at org.scalatest.MatchersHelper$.newTestFailedException(MatchersHelper.scala:160)
[info]   at org.scalatest.Matchers$ShouldMethodHelper$.shouldMatcher(Matchers.scala:6231)
[info]   at org.scalatest.Matchers$AnyShouldWrapper.should(Matchers.scala:6265)
[info]   at akka.stream.testkit.StreamTestKitSpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply$mcV$sp(StreamTestKitSpec.scala:40)
[info]   at akka.stream.testkit.StreamTestKitSpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(StreamTestKitSpec.scala:24)
[info]   at akka.stream.testkit.StreamTestKitSpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(StreamTestKitSpec.scala:24)
[info]   at org.scalatest.Transformer$$anonfun$apply$1.apply$mcV$sp(Transformer.scala:22)
[info]   at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   at org.scalatest.WordSpecLike$$anon$1.apply(WordSpecLike.scala:953)
[info]   at org.scalatest.Suite$class.withFixture(Suite.scala:1122)
[info]   at akka.stream.testkit.AkkaSpec.withFixture(AkkaSpec.scala:58)
[info]   at org.scalatest.WordSpecLike$class.invokeWithFixture$1(WordSpecLike.scala:950)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTest$1.apply(WordSpecLike.scala:962)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTest$1.apply(WordSpecLike.scala:962)
[info]   at org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
[info]   at org.scalatest.WordSpecLike$class.runTest(WordSpecLike.scala:962)
[info]   at akka.stream.testkit.AkkaSpec.runTest(AkkaSpec.scala:58)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTests$1.apply(WordSpecLike.scala:1021)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTests$1.apply(WordSpecLike.scala:1021)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:413)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]   at scala.collection.immutable.List.foreach(List.scala:381)
[info]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]   at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:390)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:427)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]   at scala.collection.immutable.List.foreach(List.scala:381)
[info]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]   at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:396)
[info]   at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:483)
[info]   at org.scalatest.WordSpecLike$class.runTests(WordSpecLike.scala:1021)
[info]   at akka.stream.testkit.AkkaSpec.runTests(AkkaSpec.scala:58)
[info]   at org.scalatest.Suite$class.run(Suite.scala:1424)
[info]   at akka.stream.testkit.AkkaSpec.org$scalatest$WordSpecLike$$super$run(AkkaSpec.scala:58)
[info]   at org.scalatest.WordSpecLike$$anonfun$run$1.apply(WordSpecLike.scala:1067)
[info]   at org.scalatest.WordSpecLike$$anonfun$run$1.apply(WordSpecLike.scala:1067)
[info]   at org.scalatest.SuperEngine.runImpl(Engine.scala:545)
```

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-nightly/2973/consoleFull

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-multi-node-nightly/3379/consoleFull

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-nightly/2974/consoleFull

**Comment by johanandren** [comment]
And three more during the nightly build

**Comment by ktoso** [comment]
Another https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/4842/consoleFull

**Comment by 2m** [comment]
Fixed in #19487
